apps/server/src/MetascanServerApp.ts

```diff
@@ -0,0 +1,56 @@
+import { NestFactory } from '@nestjs/core';
+import { NestFastifyApplication } from '@nestjs/platform-fastify';
+
+import { AppModule } from './app.module';
+
+/**
+ * App which starts the default Metascan Server Application
+ */
+export class MetascanServerApp<App extends NestFastifyApplication = NestFastifyApplication> {
+  protected app?: App;
+
+  constructor(protected readonly module: any) {}
+
+  async createNestApp(): Promise<App> {
+    const app = await NestFactory.create(AppModule);
+    await this.configureApp(app);
+    // @ts-ignore
+    return app;
+  }
+
+  async configureApp(app): Promise<void> {
+    app.enableCors({
+      allowedHeaders: 'no-cors',
```

Comment:
```suggestion
      allowedHeaders: '*',
```

Reply:
Add origin:

1. Regex for deploy previewss
2. Regex for subdomains of  meta.defiscan.live
3. meta.defiscan.live